client/app/config/index.js

```diff
@@ -72,59 +60,11 @@ function registerExtensions() {
   registerAll(context);
 }
 
-function registerServices() {
-  const context = require.context("@/services", true, /^((?![\\/.]test[\\./]).)*\.js$/);
-  registerAll(context);
-}
-
 function registerVisualizations() {
   const context = require.context("@/visualizations", true, /^((?![\\/.]test[\\./]).)*\.jsx?$/);
   registerAll(context);
 }
 
-function registerPages() {
-  const context = require.context("@/pages", true, /^((?![\\/.]test[\\./]).)*\.jsx?$/);
-  const routesCollection = registerAll(context);
-  routesCollection.forEach(routes => {
-    ngModule.config($routeProvider => {
-      each(routes, (route, path) => {
-        logger("Registering route: %s", path);
-        route.authenticated = route.authenticated !== false; // could be set to `false` do disable auth
-        if (route.authenticated) {
-          route.resolve = extend(
-            {
-              __organizationStatus: () => organizationStatus.refresh(),
-            },
-            route.resolve
-          );
-        }
-        $routeProvider.when(path, route);
-      });
-    });
-  });
-
-  ngModule.config($routeProvider => {
-    $routeProvider.otherwise({
-      resolve: {
-        // Ugly hack to show 404 when hitting an unknown route.
-        error: () => {
-          const error = { status: 404 };
-          throw error;
-        },
-      },
-    });
-  });
-}
-
 requireImages();
-registerServices();
-initAppView(ngModule);
-registerPages();
 registerExtensions();
 registerVisualizations();
```

Comment:
Probably worth switching to static imports like you did for the pages.

Reply:
Indeed :slightly_smiling_face: But it's not related to a subject of this PR, so I'll do it separately

Reply:
https://github.com/getredash/redash/issues/4565